src/Symfony/Component/Mime/Part/SMimePart.php

```diff
@@ -18,6 +18,9 @@
  */
 class SMimePart extends AbstractPart
 {
+    /** @internal */
+    public $_headers;
```

Comment:
I think it could even be private

Reply:
I've made it protected and all similar ones too.
I prefer to not make them private to be extra sure not to break inheritance.
`@internal`+protected looks safe enough to.